uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html

```diff
@@ -23,7 +23,6 @@ <h4 class="modal-title">
         autofocus
         ng-model="$ctrl.request.contact_id"
         ng-change="$ctrl.initAfterContactSelection()"
-        theme="civihr-ui-select"
```

Comment:
good questions, tried to to git bisect and changed to earlier version of civicrm, could not locate the cause of regression yet